GiniVision/Classes/Core/Helpers/GiniVisionUtils.swift

```diff
@@ -31,7 +31,7 @@ func UIImageNamedPreferred(named name: String) -> UIImage? {
  
  - returns: String resource for the given key.
  */
-func NSLocalizedStringPreferred(_ key: String, comment: String, args: CVarArg? = nil) -> String {
+func NSLocalizedStringPreferredFormat(_ key: String, comment: String) -> String {
```

Comment:
Why format? There's no format there.

Reply:
It is actually the format, because it could have external arguments, like this example: "This is an %@"